### PR TITLE
Custom SSL

### PR DESCRIPTION
**Pull request by nishanths**
Related to issue #44 

Work in progress. Tests coming soon. 👍

### CODE REVIEW

File: ssl.go [hunk 1]
```diff
@@ -1,44 +1,154 @@
...
+	ZoneID        string     `json:"zone_id"`
+	Permissions   []string   `json:"permissions"`
+	UploadedOn    time.Time  `json:"uploaded_on"`
+	ModifiedOn    time.Time  `json:"modified_on"`
+	ExpiresOn     time.Time  `json:"expires_on"`
+	KeylessServer KeylessSSL `json:"keyless_server"`
+}
+
+// ZoneCustomSSLResponse represents the response from the zone SSL details endpoint.
+type ZoneCustomSSLResponse struct {
```
Comment (elithrar) [inline ssl.go]: Ref #78 - make this private.

File: ssl.go [hunk 2]
```diff
@@ -1,44 +1,154 @@
...
+	Response
+	Result ZoneCustomSSL `json:"result"`
+}
+
+// ZoneCustomSSLsResponse represents the response from the zone SSL list endpoint.
+type ZoneCustomSSLsResponse struct {
```
Comment (elithrar) [inline ssl.go]: Ref #78 - make this private.

**Comment by elithrar** [comment]
Great! Just a couple of comments to get ahead of the work we are planning in #78. @ reply me when you push tests as I don't get a notification on new commits (just comments/issues/PRs).

**Comment by nishanths** [comment]
@elithrar fixed, and added tests :)

**Comment by jamesog** [comment]
Nice! 👍

**Comment by elithrar** [comment]
LGTM. @Tenzer ?

**Comment by Tenzer** [comment]
LGTM as well, thanks for your contribution!

**Pull request merged**